docs/example_event_dispatch.md

```diff
@@ -0,0 +1,184 @@
+---
+id: example_event_dispatch
+title: Event dispatch
+sidebar_label: Event dispatch
+---
+
+By using **trigger** [`on-event`](/docs/reference_behavior_attributes#on-event) and the **behavior action** [`dispatch-event`](/docs/reference_behavior_attributes#dispatch-event), we can communicate between different hyperview screens or between elements in the same screen.
+
+Consider a screen which shows a list of contacts along with a status.
+Another screen shows the details of a single person with an option add them to your network.
+Every time a person is added, we want the list on the first screen to show the updated status.
+
+<!-- Replace with correct gif -->
```

Comment:
is this comment still relevant? I see a few of these in the markup.

Reply:
I've removed them now.